core/src/main/java/com/linecorp/armeria/common/HttpResponseUtil.java

```diff
@@ -20,10 +20,15 @@
 
 import java.util.concurrent.CompletableFuture;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 import com.linecorp.armeria.common.util.Exceptions;
 
 final class HttpResponseUtil {
 
+    static final Logger logger = LoggerFactory.getLogger(HttpResponseUtil.class);
```

Comment:
optional) `HttpResponseUtil` seems like an package-private class which makes it susceptible to us refactoring this class without realizing that users are using this logger pattern

Reply:
Add a comment for that. 😉